src/main/groovy/net/neoforged/gradleutils/tasks/CIConfigExtractionTask.java

```diff
@@ -125,4 +130,40 @@ private String exportResource() throws Exception {
 
         return jarFolder + templateZipName;
     }
+
+    private boolean disableDeletion;
+
+    @Input
+    public boolean getDisableDeletion() {
+        return this.disableDeletion;
+    }
+
+    @Option(option = "disable-deletion", description = "Disable deletion of existing workflows")
+    public void setDisableDeletion(boolean disableDeletion) {
+        this.disableDeletion = disableDeletion;
+    }
```

Comment:
Personally, I think it'd be more... _better?_ if the option was `delete-existing` with a default value of `true`.

In any case, the description should probably note that the default behavior is to delete existing workflows.

(Though, that does bring into mind whether we should be deleting existing workflows anyway...)